kamu-adapter-http/src/ws_smart_transfer_protocol_tungstenite_client.rs

```diff
@@ -72,6 +69,45 @@ enum SmartProtocolPullClientError {
 
 /////////////////////////////////////////////////////////////////////////////////
 
+#[derive(Error, Debug)]
+enum SmartProtocolPushClientError {
```

Comment:
Same comment as for the other `SmartProtocolPushServerError` - hard to imagine how clients would handle this error.